README.md

```diff
@@ -264,6 +264,8 @@ process.
 For more details about how Bats evaluates test files, see [Bats Evaluation
 Process][bats-eval] on the wiki.
 
+For sample test files, see [examples](/examples).
```

Comment:
The directory `/docs/examples` seems like a better home for this content.

```suggestion
For sample test files, see [examples](/docs/examples).
```

Include `examples/README.md` which at the very least should contain a link to the [Wiki Examples page](https://github.com/bats-core/bats-core/wiki/Examples).

```markdown
# Examples

See the [bats-core wiki][examples] for more details

[examples]: (/bats-core/bats-core/wiki/Examples)
```